### PR TITLE
ddl, metrics: add worker info to the log

### PR DESCRIPTION
**Pull request by zimulala**
## What have you changed? (mandatory)
Add worker info to the log.
[ddl] -> [ddl-worker 0, tp general]

## What is the type of the changes? (mandatory)
- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested? (mandatory)

No.
## Does this PR affect documentation (docs/docs-cn) update? (mandatory)

No.
## Does this PR affect tidb-ansible update? (mandatory)

No.

## Does this PR need to be added to the release notes? (mandatory)

No.

### CODE REVIEW

File: ddl/ddl_worker.go [hunk 1]
```diff
@@ -98,20 +98,20 @@ func (w *worker) typeStr() string {
...
 	w.wg.Wait()
-	log.Infof("[ddl] close DDL worker %s", w)
+	log.Infof("[ddl-%s] close DDL worker", w)
 }
 
 // start is used for async online schema changing, it will try to become the owner firstly,
 // then wait or pull the job queue to handle a schema change job.
 func (w *worker) start(d *ddlCtx) {
-	log.Infof("[ddl] start DDL worker %s", w)
+	log.Infof("[ddl-%s] start DDL worker", w)
```
Comment (winkyao) [inline ddl/ddl_worker.go]: I think keeping the previous format is better. What is the difference?

Comment (zimulala) [reply ddl/ddl_worker.go]: [ddl] -> [ddl-worker 0, tp general]

Comment (winkyao) [reply ddl/ddl_worker.go]: So what is the problem with the previous log?

Comment (zimulala) [reply ddl/ddl_worker.go]: Other places need to add `[ddl-worker 0, tp general]`. I want to unify the log, so do it.

**Comment by zimulala** [comment]
PTAL @winkyao @tiancaiamao @coocood @ciscoxll @crazycs520

File: metrics/ddl.go [hunk 2]
```diff
@@ -91,7 +91,7 @@ var (
 		}, []string{LblType, LblResult})
 
 	CreateDDLInstance = "create_ddl_instance"
-	CreateDDLWorker   = "create_ddl_worker"
+	CreateDDL         = "create_ddl"
```
Comment (ciscoxll) [inline metrics/ddl.go]: Why change this.

**Comment by ciscoxll** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**